packages/odyssey-design-tokens/config.json

```diff
@@ -0,0 +1,24 @@
+{
+  "source": ["src/*.json"],
+  "platforms": {
+    "js": {
+      "transformGroup": "js",
+      "files": [
+        {
+          "destination": "dist/js/tokens.js",
```

Comment:
so no TS type annotations for this? can we have these output alongside the javascript please this will be useful to teams